st2common/st2common/openapi.yaml

```diff
@@ -2069,6 +2074,72 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /api/v1/rbac/role_assignments:
```

Comment:
Yeah, for now we are just adding "additive" non-breaking changes to v1 and we don't really have other versioning scheme in place.

If and when we start changing API responses for existing endpoints, we will probably move to v2 or similar.

Oh, and we add "experimental" non-stable stuff to /exp/.